apps/docs/content/components/content/Avatar.mdx

```diff
@@ -1,48 +1,105 @@
 ---
-title: Avatar (WIP)
-description: TBD
+title: Avatar
+description: Avatars are used to represent a user, team or another entity. They are often paired with text where room is available. If an image is not provided or fails to load, the initials of the name will be displayed instead.
 category: "content"
 links:
-    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/TBD/src/TBD.tsx
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/Avatar/src/index.tsx
 ---
 
+<Example src="avatar/docs/preview" isOpen />
+
 ## Props
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+<PropTable component="Avatar" />
+
+<FeatureFlag flag="next">
+    ## Guidelines
 
-## Guidelines
+    TODO: If we have some guidelines about this component's usage
 
-TODO: If we have some guidelines about this component's usage
+    ### Accessibility ?
 
-### Accessibility ?
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
-TODO: If we have some guidelines about this component and accessibility
+<FeatureFlag flag="rc">
+    ## Anatomy
 
-## Anatomy
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+    ### Concepts
 
-### Concepts
+    TODO: links to related concepts
+</FeatureFlag>
 
-TODO: links to related concepts
+## Examples
 
-### Composed Components
+### Local image
 
-TODO: links to related components (most likely all component which are used by the slots)
+It would normally be imported from a file like this: `import Frog from "./hadfield.png";` then used as the src.
 
-## Examples
+<Example src="avatar/docs/localImage" />
+
+### Remote image
+
```

Comment:
We usually add a short sentence adding more context to each examples.

Reply:
some of them only had a description like: "An Avatar with a remote image."
I think it makes sense not to add these, unless you can come up with something better. Sorry, they weren't very good. Not much to say, I think.

Reply:
This could be done in a v2, for consistency reasons I think we should have small sentences. Like : An Avatar can fetch a remote image.